packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -185,6 +185,11 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    */
   keytipProps?: IKeytipProps;
 
+  /**
+   * Optional mode indicates if arrow keys are valid input when disabled.
+   */
+  allowArrowKeyInputWhenDisabled?: boolean;

```

Comment:
the name of this prop isn't clear

Reply:
What would be more clear?

Reply:
maybe `cancelKeysWhenDisabled` is a little more clear because that's exactly what's happening